honeybee_radiance_command/rfluxmtx.py

```diff
@@ -0,0 +1,139 @@
+"""rfluxmtx command"""
+
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+from .options.rfluxmtx import RfluxmtxOptions, RfluxmtxControlParameters
+
+
+# octree to octree and so on
+class Rfluxmtx(Command):
+    """
+    Rfluxmtx command.
+
+    Rfluxmtx computes the flux transfer matrices for a RADIANCE scene.
+
+    """
+
+    __slots__ = ('_input', "_sensors", "_sender", "_octree", "_receivers", "_system")
+
+    def __init__(self, options=None, output=None, sensors=None, sender=None, receivers=None, system=None,
+                 octree=None):
+        """Initialize Command."""
+        Command.__init__(self, output=output)
```

Comment:
What if you subclass from Rcontrib command here? That will re-use octree and sensors from Rcontrib/Rtrace.

Reply:
We could do that, but then we'd have to do some code gymnastics by trying to re-implement methods (including setters and validate). Both octree and sensors are required inputs for rcontrib but they are optional for rfluxmtx (through - and -i options respectively).